javascript/packages/orchestrator/src/network.ts

```diff
@@ -95,16 +95,23 @@ export class Network {
   wasRunning: boolean;
```

Comment:
```suggestion
  wasRunning: boolean = false;
```

Can do the same for launched above

Reply:
```suggestion
  wasRunning = false;
```
The type is trivially inferred from the boolean literal thus the type annotation can be removed